src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -190,7 +190,16 @@ private static string FindBestVersionMatch(VersionRange versionRange, IEnumerabl
                 return version;
             }).Where(v => v != null);
 
-            return versionRange.FindBestMatch(bundleVersions)?.ToString();
+            var sortedBundleVersions = bundleVersions.OrderByDescending(version => version.Version);
+            for (int i = 0; i < sortedBundleVersions.Count(); i++)
```

Comment:
As we are using nuget version semantics, it is better to not change how a version range is interpreted. Why not just enforce [2.*, 3.0.0) format if users need to get the latest version available?

Reply:
We're using nuget APIs and applying the version range, which still applies, but the version selection behavior is not tied to them (this is not documented or advertised as being NuGet version selection).

Reply:
Ok. We do have to document what version ranges mean similar to https://docs.microsoft.com/en-us/nuget/reference/package-versioning#version-ranges-and-wildcards